### PR TITLE
circle ci

### PR DESCRIPTION
**Pull request by DonBrinn**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 dirversion: 2.1
```
Comment (bemailloux) [inline .circleci/config.yml]: I haven't seen `dirversion` before - is there a difference from `version`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
 jobs:
-  test_js:
+  build:
```
Comment (bemailloux) [inline .circleci/config.yml]: This is cool - I didn't know you could build without workflows

File: .circleci/config.yml (hunk 1 above)
Comment (DonBrinn) [reply .circleci/config.yml]: I copied this from insights-self-serve-reporting.  I don't know.

Comment (bemailloux) [reply .circleci/config.yml]: Huh that's... interesting - here's what I have for the beginning of insights-self-serve-reporting's circleci:

```version: 2.1
orbs:
  slack: circleci/slack@3.4.0

commands:
...
```

File: .circleci/config.yml (hunk 2 above)
Comment (DonBrinn) [reply .circleci/config.yml]: Oops.  This was a typo on my part, I thought I was typing "dir" into a command prompt.

File: .circleci/config.yml (hunk 1 above)
Comment (DonBrinn) [reply .circleci/config.yml]: Oops. This was a typo on my part, I thought I was typing "dir" into a command prompt.

**Pull request merged**